src/polyswarm_api/__init__.py

```diff
@@ -451,14 +461,23 @@ def _fix_result(self, result):
         response['search'] = query
         return response
 
-    async def rescan_hash(self, to_rescan, hash_type='sha256'):
+    async def rescan_hash(self, to_rescan, hash_type=None):
         """
         Start a rescan for single hash using the PS API asynchronously.
 
         :param to_rescan: hash of the file to rescan
-        :param hash_type: Hash type [sha256|sha1|md5]
+        :param hash_type: Hash type [default:sha256|sha1|md5]
         :return: JSON report file
         """
+        if not hash_type or hash_type == 'sha1' or \
```

Comment:
Same thing here, we shouldn't have to add checks to this every time we add a hash

Reply:
done